### DISCUSSION TITLE
Feature request: Search by Twitter handle

### DISCUSSION THREAD

**Discussion by jschuur** [community/community#27275] [Apps, API and Webhooks]
I don't think it's supported in the [search syntax](https://docs.github.com/en/search-github/getting-started-with-searching-on-github/understanding-the-search-syntax) (I tried variations of 'twitter:joostschuur' e.g.), but it would be useful if the search API let you find users by their Twitter profile name (specifically, the `twitter_username` property on the GitHub user).

Use case would be to create a tool that lets you find the GitHub profiles of the people you follow on Twitter.

Bonus points for batch lookups :)

**Comment by jfullstackdev** [comment]
it's available on the endpoint

<https://api.github.com/users/jdevstatic>

![image](https://user-images.githubusercontent.com/47092464/181255262-c4be703c-301e-4c8e-b959-0f012b15d44f.png)

`"twitter_username": "jdevstatic"`

you can create scripts to achieve as what you want to do, say you want to check whether you follow them both on Twitter and GitHub

**Reply by jschuur** [reply]
> it's available on the endpoint
> 
> https://api.github.com/users/jdevstatic

Indeed, once I know their GitHub username, I can get their Twitter handle, but for someone on Twitter trying to find what people they follow are on GitHub, I can't do a GitHub search by Twitter handle to find out their GitHub username in the first place.

My suggestion was for the Twitter handle to become a filterable criteria in the GitHub Search API.

I can try the same username on GitHub than Twitter and then look up their GitHub `twitter_username` property to see if it matches, but that isn't 100% reliable.

**Reply by jfullstackdev** [reply]
also, maybe you can dig farther on Twitter side, say, users you follow and get the data through its API and that's it, if there is a GitHub handle
